lib/attributes/utils/replace_attribute.h

```diff
@@ -37,5 +37,6 @@ HandleResult handleCXXRecord(SessionStage&,
 HandleResult handleTranslationUnit(SessionStage& s,
                                    const clang::TranslationUnitDecl& decl,
                                    std::vector<std::string_view> headers,
+                                   std::vector<std::string_view> defines,
```

Comment:
Please make it empty by default as for namespaces

Reply:
While the Default initializer was addressed.
I still think that the calls are better be addressed too.